library/src/methods/getMetadata/getMetadata.test.ts

```diff
@@ -0,0 +1,49 @@
+import { describe, expect, test } from 'vitest';
+import * as v from '../../index.ts';
+
+describe('getMetadata', () => {
```

Comment:
Perhaps we should have a test, which tests with nested object?

Reply:
not sure what you mean by nested object, sorry

Reply:
```
v.pipe(
  v.object({
    name: v.pipe(v.string(), v.title("The Name")0
  }),
  v.title("An object")
);
```

Or does this just get metadata of a single field?

Reply:
this just gets the metadata of the single schema, not any nested schemas - you'd need to do `v.getMetadata(schema.pipe[0].entries.name)`